Files.Launcher/Program.cs

```diff
@@ -422,11 +422,13 @@ private static async Task ParseFileOperationAsync(AppServiceRequestReceivedEvent
                     });
                     break;
 
-                case "MoveToBin":
+                case "DeleteItem":
                     var fileToDeletePath = (string)args.Request.Message["filepath"];
+                    var permanently = (bool)args.Request.Message["permanently"];
                     using (var op = new ShellFileOperations())
                     {
-                        op.Options = ShellFileOperations.OperationFlags.AllowUndo | ShellFileOperations.OperationFlags.NoUI;
+                        op.Options = ShellFileOperations.OperationFlags.NoUI;
+                        if (!permanently) op.Options |= ShellFileOperations.OperationFlags.AllowUndo;
```

Comment:
Please, always check your code is match to our [Code style](https://github.com/files-community/Files/wiki/Code-style).